accord-core/src/test/java/accord/impl/basic/NodeSink.java

```diff
@@ -33,11 +44,20 @@
 import accord.messages.Reply;
 import accord.messages.ReplyContext;
 import accord.messages.Request;
+import com.google.common.collect.ImmutableSet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static accord.impl.basic.Packet.SENTINEL_MESSAGE_ID;
+import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class NodeSink implements MessageSink
 {
+    private static final boolean DEBUG = false;
+    private enum Action {DELIVER, DROP, DROP_PARTITIONED, FAILURE}
```

Comment:
in follow up patches I add client actions as well, so we get a more complete view